helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -111,7 +111,7 @@
   public static final int DEFAULT_MAX_DISCONNECT_THRESHOLD = 600; // Default to be a large number
   private static final int DEFAULT_WAIT_CONNECTED_TIMEOUT = 10 * 1000;  // wait until connected for up to 10 seconds.
 
-  protected final String _zkAddress;
+  protected String _zkAddress;
```

Comment:
No need to remove final right?

Reply:
Now that we've changed the design for HelixManager constructors to always accept zkAddress, we could bring the `final` keyword back.